conf/application.conf

```diff
@@ -78,7 +78,7 @@ play.i18n.langs = ["en"]
 # you may need to define a router file `conf/my.application.app.routes`.
 # Default to Routes in the root package (and conf/app.routes)
 # !!!WARNING!!! DO NOT CHANGE THIS ROUTER
-play.http.router = prod.Routes
+play.http.router = testOnlyDoNotUseInAppConf.Routes
```

Comment:
This shouldn't be checked in

Reply:
Thanks, Robertos. This was to test the Swagger endpoint

Reply:
I'm going to update the README to suggest people run the project with his command:

`sbt run "9051 -Dplay.http.router=testOnlyDoNotUseInAppConf.Routes"`